tests/provision/provisioning_test.go

```diff
@@ -0,0 +1,520 @@
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/volume"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultVstate = 1
+)
+
+func TestVps(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_basic.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Provisioning", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+// This test performs VolumePlacementStrategy's replica affinity  of application
+// volume
+var _ = Describe("{ReplicaAffinity}", func() {
+	It("has to schedule app and verify the volume replica affinity", func() {
```

Comment:
after `It` we need to collect bundle in case of failure, check the example here https://github.com/portworx/torpedo/blob/5644fcce6fb4d4f33db7cb767a2c6317cb26e3bc/tests/basic/basic_test.go#L104

Reply:
Incorporated the changes